modules/texture.lobster

```diff
@@ -45,3 +45,39 @@ def loadtexturecached(name, tf:texture_format):
             return from program
 
 def loadtexturecached(name): return loadtexturecached(name, texture_format_none)  // FIXME: default args.
+
+// Pass nil for tex to have it created. Recreates tex if the wrong size. Renders f. Returns tex.
+// Depth texture automatically created if depth is true and depthtex is nil.
+def render_to_texture(tex, size, depth, depthtex, texture_format, f):
+    if !tex || gl_texture_size(tex) != size:
+        tex = gl_create_blank_texture(size, color_black, texture_format)
```

Comment:
Later realized: we should change the initial color her to `color_clear`:
- it's not possible to `gl_clear(color_clear) // clear to transparent`
- but you can clear to black

Reply:
it's kinda best the user calls `gl_clear` themselves, since it is expensive, and don't want to do it twice.. in fact sometimes its ok to not do it at all.